src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -50,8 +55,8 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
         </View>
         <View style={styles.containerButton}>
           <Pressable style={styles.followAction(32, 32)}>
-            <Animated.View style={{position: 'absolute', opacity: followIconFadeAnimation}}>
-              <MemoIc_Checklist width={32} height={32} />
+            <Animated.View style={{position: 'absolute', top: 4, opacity: followIconFadeAnimation}}>
+              <MemoIc_Checklist width={32} height={32} color={isAnon ? '#107793' : '#4782D7'} />
```

Comment:
mas @GarryLaly please use colors only from `/src/utils/theme/index.js`, do not use hardcoded string.

Reply:
Fixed